### PR TITLE
shorten rownames (extract species name)

### PR DESCRIPTION
**Pull request by LisaHollstein**

### CODE REVIEW

**Comment by colindaven** [comment]
Hi Lisa, this code is where Sophia shortened the name briefly. 

https://github.com/MHH-RCUG/haybaler/blob/d8343def80d6966abf87d58e52f7f915929da760/haybaler.py#L47

I'm wondering if it might be better to do this work at that point, i.e. in Python pandas ? We can try both of course or work through it in more detail before making a decision.

**Comment by LisaHollstein** [comment]
Okay, I will try to do it at that point

**Comment by colindaven** [comment]
Perhaps we can include this one as well in a subfolder as an accessory script, since it is already complete ?

File: haybaler.py [hunk 1]
```diff
@@ -140,6 +140,59 @@ def exclude_taxa(file, path, taxa_to_exclude):
...
+    save_name = output_path + "/" +  col + "_" + output_file
+    save_name = save_name.split(".")[0] + "_short.csv"
+
+    index = short.index
+    if index.is_unique:  # only saved if all row names are unique
+        short.to_csv(save_name, sep='\t')
+    # if not all row names are unique: for the read count csv the values of the non unique rows are summed
+    elif col == "read_count":
+        short = short.groupby("species").sum()
+        short.to_csv(save_name, sep='\t') # save read count table
```
Comment (LisaHollstein) [inline haybaler.py:176]: Changes the row order

Maybe don't write any short csv if index is not unique?

**Comment by colindaven** [comment]
bumped version, added Lisa

**Comment by colindaven** [comment]
Happy, can I  merge and test ?

**Comment by LisaHollstein** [comment]
Yes  👍

**Pull request merged**